src/Appwrite/Utopia/Response/Model/Document.php

```diff
@@ -43,16 +43,16 @@ public function __construct()
                 'example' => '5e5ea5c15117e',
             ])
             ->addRule('$createdAt', [
-                'type' => self::TYPE_INTEGER,
-                'description' => 'Document creation date in Unix timestamp.',
-                'default' => 0,
-                'example' => 1592981250,
+                'type' => self::TYPE_DATETIME,
+                'description' => get_class() . ' creation date in Datetime',
```

Comment:
We probably should be more detailed here, on what Format is actually returned 👍🏻

Reply:
The format we can show as an example?